pom.xml

```diff
@@ -2511,8 +2523,8 @@
   <profiles>
 
     <!--
-      This profile is enabled automatically by the sbt built. It changes the scope for the guava
-      dependency, since we don't shade it in the artifacts generated by the sbt build.
+      This profile is enabled automatically by the sbt build. It changes the scope for shaded
```

Comment:
I wonder if this profile is even necessary, given it does not include jetty which is also "provided" by default... but I'm too lazy to do history spelunking at the moment.